packages/dapp/src/components/VC/tabs/FormatedTab.tsx

```diff
@@ -51,40 +57,43 @@ const FormatedTab = ({
   const types = convertTypes(vc.data.type);
 
   const expDate = vc.data.expirationDate
-    ? `Expires on ${new Date(
```

Comment:
We could use [Luxon](https://moment.github.io/luxon/#/?id=luxon) for Date handling